src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,160 @@
+import { useSearchParams } from 'react-router-dom';
+import { Person } from '../../types';
+import { PersonLink } from '../PersonLink/PersonLink';
+import cn from 'classnames';
+
+/* eslint-disable jsx-a11y/control-has-associated-label */
+
+type Props = {
+  people: Person[];
+};
+
+export const PeopleTable: React.FC<Props> = ({ people }) => {
+  const [searchParams, setSearchParams] = useSearchParams();
+  const sortField = searchParams.get('sort') || null;
+  const sortOrder = searchParams.get('order') || 'asc';
+
+  const peopleWithParents = people.map(person => {
+    const mother = people.find(mom => mom.name === person.motherName);
+    const father = people.find(dad => dad.name === person.fatherName);
+
+    return { ...person, mother, father };
+  });
+
+  const handleSort = (field: string) => {
+    if (sortField === field && sortOrder === 'asc') {
+      return setSearchParams({ sort: field, order: 'desc' });
+    }
+
+    if (sortField === field && sortOrder === 'desc') {
+      return setSearchParams({});
```

Comment:
When resetting the sort parameters, consider explicitly setting `order` to `asc` or removing both `sort` and `order` to ensure the default sorting state is clear.